### ISSUE TITLE
Mark Seen option does not accept an array

### ISSUE THREAD

**Issue by araines** [GetStream/stream-php#27]
More of an inconsistency than anything else, but when passing options to a notification feed, the documentation states the same for both `mark_seen` and `mark_read`:

```
NAME	    DESCRIPTION
mark_read	Comma separated list of activity ids to mark as read, or boolean true
mark_seen	Comma separated list of activity ids to mark as seen, or boolean true
```

And the code example given shows this being achieved with an array rather than a string:

```
# Mark some activities as read
$options = array('mark_read' => array('activity_id_1', 'activity_id_2'));
$results = $notification_feed->getActivities(0, 10, $options);
```

This works fine for `mark_read` but does not work for `mark_seen`.  I believe both should behave the same way.

**Comment by tbarbugli** [comment]
fixed and deployed 2.2.3 please let us know if this you are still experiencing issues with this.

**Comment by araines** [comment]
Many thanks
